registries/_format/binary.md

```diff
@@ -12,7 +12,7 @@ layout: default
 
 Base type: `{{ page.base_type }}`.
 
-The `{{page.slug}}` format represents any sequece of octets. This format entry is to ensure future versions of OpenAPI maintain compatibility with [OpenAPI 3.0.x](https://spec.openapis.org/oas/v3.0.0).
+The `{{page.slug}}` format represents any sequence of octets. This format entry is to ensure future versions of OpenAPI maintain compatibility with [OpenAPI 3.0.x](https://spec.openapis.org/oas/v3.0.0).
```

Comment:
I think it would be worth noting that in OAS 3.1 this is better written as `"contentMediaType": "application/octet-stream"` without a `"type"` (because binary data is outside of the JSON Schema data model).

Reply:
right, except when you have scenarios when API return a mix of structured and unstructured data and they are not using a multipart response/body.
A good example of that is a scenario of search result, where both the structured payload and a thumbnail image are returned in a single call.
So should we add a not to suggest people using a different media type is better when the full payload is binary?

Reply:
I'm not sure I follow.  What is the media type of the response in your example?  You cannot embed raw binary in `application/json`.

Reply:
@baywet this comment is attached to the `"format": "binary"` definition but your example uses `"format": "base64url"` so now I am even more confused.

Reply:
Thanks, this clarifies confusion on my end, kiota is loosing precision on that front by mapping binary, base64 and base64url to a mushed "binary" format which it shouldn't be doing since the serialization/deserialization is going to be different (one is straight stream and invalid for properties, others are stream + serialization logic).

We'll have to retro-fit that aspect.

The example provided above applies to base64 and base64url only then.
I'll update this one shortly to provide additional guidance. Let me know what you think.

Reply:
https://github.com/OAI/OpenAPI-Specification/pull/3167/commits/f45235b8c9c8860c95197eeb8e700af1f2072a0e